.github/workflows/ci_workflows.yml

```diff
@@ -120,16 +124,30 @@ jobs:
       test_command: pytest --pyargs erfa
       targets: |
         # Linux wheels
-        - cp*-manylinux_i686
```

Comment:
Note that I also dropped 32-bit wheels here. Numpy doesn't provide 32-bit anymore.